package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ibm.github.io",
+  "description": "ibm.github.io",
+  "author": "IBM",
+  "version": "0.0.1",
+  "devDependencies": {
+    "jshint": "^2.9.5"
+  },
+  "scripts": {
+    "test": "./node_modules/jshint/bin/jshint orgs.js"
```

Comment:
I think you can just run "jshint orgs.js" but this is OK

Reply:
let me try